src/bundle/Resources/public/js/scripts/helpers/table.helper.js

```diff
@@ -0,0 +1,19 @@
+(function (global, doc, eZ, $) {
+    parseCheckbox = (checkboxSelector) => {
+        doc.querySelectorAll(checkboxSelector).forEach((checkboxNode) => {
+            const parentRow = checkboxNode.closest('tr');
+            const activeClass = parentRow.classList.contains('c-table-view-item') ? 'c-table-view-item--active' : 'ez-table__row--active';
```

Comment:
I think the activeClass should be passed as an argument, not checked inside this method. Thank to that we would have a more generic solution.
What if there will be another active class?